tests/accesscontrol/tests/access_control_pod_role_bindings.go

```diff
@@ -0,0 +1,131 @@
+package accesscontrol
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/accesscontrol/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/pod"
+)
+
+func setupInitialRbacConfiguration() {
+	By("Create service account")
+
+	err := globalhelper.CreateServiceAccount(tsparams.TestServiceAccount, tsparams.TestAccessControlNameSpace)
+	Expect(err).ToNot(HaveOccurred())
+
+	err = globalhelper.CreateRole(tsparams.TestRoleName, tsparams.TestAccessControlNameSpace)
+	Expect(err).ToNot(HaveOccurred())
+
+	err = globalhelper.CreateRoleBindingWithServiceAccountSubject(tsparams.TestRoleBindingName, tsparams.TestRoleName,
+		tsparams.TestServiceAccount, tsparams.TestAccessControlNameSpace)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+var _ = Describe("Access-control pod-role-bindings,", func() {
+
+	BeforeEach(func() {
+		By("Clean namespace before each test")
+		err := namespaces.Clean(tsparams.TestAccessControlNameSpace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = namespaces.Clean(tsparams.TestAnotherNamespace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+
+		setupInitialRbacConfiguration()
+	})
+
+	It("one pod with valid role binding", func() {
+		By("Define pod")
+
+		testPod := pod.DefinePod(tsparams.TestPodName, tsparams.TestAccessControlNameSpace,
+			globalhelper.Configuration.General.TestImage, tsparams.TestDeploymentLabels)
+
+		pod.RedefineWithServiceAccount(testPod, tsparams.TestServiceAccount)
+
+		err := globalhelper.CreateAndWaitUntilPodIsReady(testPod, tsparams.Timeout)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start pod-role-bindings")
+		err = globalhelper.LaunchTests(
+			tsparams.TnfPodRoleBindings,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TnfPodRoleBindings,
+			globalparameters.TestCaseSkipped)
```

Comment:
Should this actually be globalparameters.TestCasePassed? When I ran these on my cluster it looks like this case passed and I got "msg: "test case \"access-control-pod-role-bindings\" is not in expected \"skipped\" state in junit report." as an error.